packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -17,6 +17,7 @@ import styles from './Menu.module.scss';
 
 const cx = classNames.bind(styles);
 const menuHeaderId = `terra-menu-headertitle-${uuidv4()}`;
+const menuTopHeaderId = `terra-menu-headertitle-${uuidv4()}`;
```

Comment:
```suggestion
const menuTopHeaderId = `terra-menu-headertitle-${uuidv4()}`;
```
since only one header is displayed at any given moment of time `menuHeaderId` can be used for both header !!

Reply:
https://github.com/cerner/terra-framework/pull/1649/commits/f3c198c07d05ed1fe309e66c569eac8d214ed205